bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/TTWW_5f_LO/TTWW_5f_LO_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm-no_b_mass
+generate p p > t t~ w+ w-
```

Comment:
I would suggest to use madspin for top and w decay instead of using pythia.

Reply:
set ms_dir ./madspingrid
set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
set max_running_process 1
decay t > w+ b, w+ > all all
decay t~ > w- b~, w- > all all
decay w+ > all all
decay w- > all all
launch


this madspin card is fine?

Reply:
"Nevents_for_max_weigth" is now "Nevents_for_max_weight"